analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.cs

```diff
@@ -69,6 +69,18 @@ public static void WithUnaryPrefix()
             result = bitMask | + + +zero;  // FN - Unary Operator is not supported
         }
 
+        public void AfterALoop()
+        {
+            var length = 0x80;
+            var bytesRequired = 0;
+            while (length > 0)
+            {
+                length >>= 8;
+                bytesRequired++;
+            }
+            _ = bytesRequired | 0x80; // Noncompliant, FP See: https://github.com/SonarSource/sonar-dotnet/issues/6326
```

Comment:
Let's add also `++bytesRequired`, `--bytesRequired`, `bytesRequired--` so we don't forget about those

Reply:
..they'll need independent variables